poetry.lock

```diff
@@ -1054,18 +1042,18 @@ django-jquery-js = "*"
 
 [[package]]
 name = "django-guardian"
-version = "2.4.0"
-description = "Implementation of per object permissions for Django."
+version = "3.0.0"
```

Comment:
why not 3.0.1?

Reply:
Wasn't available in pypi when I did the upgrade